### PR TITLE
Reorganise forced transactions

### PR DESCRIPTION
**Pull request by sin6pi7**

### CODE REVIEW

File: packages/backend/src/peripherals/database/migrations/014_forced_transactions.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+because migrations are only run once on the production server. 
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+import { up as createForcedTransactionEvents } from './012_forced_transaction_events'
```
Comment (sz-piotr) [inline packages/backend/src/peripherals/database/migrations/014_forced_transactions.ts:16]: Is there a reason up is used, but down is reimplemented with `dropTableIfExists`?

Comment (sin6pi7) [reply packages/backend/src/peripherals/database/migrations/014_forced_transactions.ts:16]: Good catch - removed.

File: packages/backend/src/peripherals/database/ForcedTransactionsRepository.ts [hunk 2]
```diff
@@ -1,351 +1,150 @@
...
+export class ForcedTransactionsRepository extends BaseRepository {
+  constructor(knex: Knex, logger: Logger) {
+    super(knex, logger)
+    this.getAll = this.wrapGet(this.getAll)
+    this.getLatest = this.wrapGet(this.getLatest)
+    this.getIncludedInStateUpdate = this.wrapGet(this.getIncludedInStateUpdate)
+    this.getAffectingPosition = this.wrapGet(this.getAffectingPosition)
+    this.getAffectingPosition = this.wrapGet(this.getAffectingPosition)
+    this.findByHash = this.wrapFind(this.findByHash)
+    this.deleteAll = this.wrapDelete(this.deleteAll)
```
Comment (sz-piotr) [inline packages/backend/src/peripherals/database/ForcedTransactionsRepository.ts:121]: What about `add` and `markAsMined`?

File: packages/backend/src/peripherals/database/migrations/014_forced_transactions.ts (hunk 1 above)
Comment (sz-piotr) [reply packages/backend/src/peripherals/database/migrations/014_forced_transactions.ts:16]: Why not just import down too?

Comment (sin6pi7) [reply packages/backend/src/peripherals/database/migrations/014_forced_transactions.ts:16]: Another good catch :D

**Review by sz-piotr — APPROVED** [review]

**Pull request merged**